test/junit/scala/collection/immutable/ListTest.scala

```diff
@@ -119,16 +119,21 @@ class ListTest extends AllocationTest {
     // - List.ReusableCBF
     // - collection.immutable.Seq.ReusableCBF
     // - collection.Seq.ReusableCBF
+    // - collection.immutable.LinearSeq.ReusableCBF
+    // - collection.LinearSeq.ReusableCBF
     // are distinct all generate List instances ( and that other don't behave the same way)
     // If this changes then (at least) the method List#isLikeListReusableCBF should be changed accordingly
 
-    assertNotSame(List.ReusableCBF, collection.immutable.Seq.ReusableCBF)
-    assertNotSame(List.ReusableCBF, collection.Seq.ReusableCBF)
-    assertNotSame(collection.immutable.Seq.ReusableCBF, collection.Seq.ReusableCBF)
+    val allCompatibleCFBs = List[CanBuildFrom[_,_,_]](List.ReusableCBF,
```

Comment:
CFB -> CBF

Reply:
haha. Must teach Intellij to correct more of my misspellings

Reply:
fixed now